sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/src/main/java/com/azure/spring/sample/feature/management/web/DisabledFeaturesHandler.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.example;
+package com.azure.spring.sample.feature.management.web;
```

Comment:
Let's change `feature.management` to `featuremanagement`

Reply:
all right